warehouse/admin/views/verdicts.py

```diff
@@ -26,23 +31,23 @@
     uses_session=True,
 )
 def get_verdicts(request):
-    try:
-        page_num = int(request.params.get("page", 1))
-    except ValueError:
-        raise HTTPBadRequest("'page' must be an integer.") from None
-
-    verdicts_query = request.db.query(MalwareVerdict).order_by(
-        MalwareVerdict.run_date.desc()
+    result = {}
+    result["check_names"] = set(
+        [name for (name,) in request.db.query(MalwareCheck.name)]
     )
+    result["classifications"] = set([c.value for c in VerdictClassification])
```

Comment:
Ultra nit, but I _think_ you can do something like `VerdictClassification.__members__.values()` to avoid comprehensions here.

Reply:
This doesn't do what I would hope:
  
    >>> VerdictClassification.__members__.values()
    odict_values([<VerdictClassification.Benign: 'benign'>, <VerdictClassification.Indeterminate: 'indeterminate'>, <VerdictClassification.Threat: 'threat'>])

Reply:
Ah, boo. Ignore this then!